Partially revert r6010 to avoid trashing the .desktop.in (due to missing '-c').

--- origsrc/waf-1.5.8/wafadmin/Tools/intltool.py	2009-06-09 18:17:29.000000000 -0500
+++ src/waf-1.5.8/wafadmin/Tools/intltool.py	2009-07-20 00:03:37.017116700 -0500
@@ -40,7 +40,7 @@ def iapply_intltool_in_f(self):
 		cache = getattr(self, 'intlcache', '.intlcache')
 		self.env['INTLCACHE'] = os.path.join(self.path.bldpath(self.env), podir, cache)
 		self.env['INTLPODIR'] = podirnode.srcpath(self.env)
-		self.env['INTLFLAGS'] = getattr(self, 'flags', ['-q', '-u', '-c'])
+		self.env['INTLFLAGS'] = getattr(self, 'flags', None)
 
 		task = self.create_task('intltool')
 		task.set_inputs(node)
@@ -96,7 +96,7 @@ def apply_intltool_po(self):
 
 Task.simple_task_type('po', '${POCOM} -o ${TGT} ${SRC}', color='BLUE', shell=False)
 Task.simple_task_type('intltool',
-	'${INTLTOOL} ${INTLFLAGS} ${INTLCACHE} ${INTLPODIR} ${SRC} ${TGT}',
+	'${INTLTOOL} ${INTLFLAGS} -q -u -c ${INTLCACHE} ${INTLPODIR} ${SRC} ${TGT}',
 	color='BLUE', after="cc_link cxx_link", shell=False)
 
 def detect(conf):
